crates/re_viewer/src/web.rs

```diff
@@ -100,6 +104,15 @@ fn create_app(cc: &eframe::CreationContext<'_>, url: Option<String>) -> crate::A
         egui_ctx.request_repaint_after(std::time::Duration::from_millis(10));
     });
 
+    let mut app = crate::App::new(build_info, &app_env, startup_options, re_ui, cc.storage);
+
+    if url.is_none() && !fallback_to_ws {
+        // We didn't get a `url` to connect to, and we also aren't supposed to
+        // start the WebSocket client, so just show the welcome screen.
+        return app;
+    }
+
+    let url = url.unwrap_or_else(|| get_url(&cc.integration_info));
```

Comment:
This looks wrong. `fallback_to_ws` will now affect wether or not we check for the `?url=…` query in `get_url`

Reply:
I wonder if we actually ever hit the `get_url` path, though?